testutil/daemon/daemon.go

```diff
@@ -721,8 +719,3 @@ func cleanupRaftDir(t testing.TB, rootPath string) {
 		}
 	}
 }
-
-// ImageService returns the Daemon's ImageService
-func (d *Daemon) ImageService() *images.ImageService {
```

Comment:
Should we keep this? (as it's exported)

Reply:
I was nervous about that, but was told it didn't matter. Sadly the point was to get rid of the dependency on `images` which was causing buildkit to need `btrfs.h` and anything else required for image storage so it really had to go.